src/utils/index.ts

```diff
@@ -341,6 +341,21 @@ const copyToClipboard = async (value: string, text?: string) => {
   });
 }
 
+const responseFileType = (response: any, fileName: string) => {
```

Comment:
We can merge these two functions. responseFileType anyways is not generating any meaning. You can name it like saveDataFile.